api/filtering/filtering.py

```
@@ -435,7 +435,13 @@ def query_filters(
     if provider_id:
         query_filters += ({"provider_id": {"eq": provider_id.casefold()}},)
     if updated_start or updated_end:
-        query_filters += _build_modified_on_filter(updated_start, updated_end)
+        # Need to check if date is less then 1970, since ElasticSearch doesn't
+        # allow timestamps before above this year. See ticket #ESSNTL-4458
+        if parser.isoparse(updated_start).year > 1970 or parser.isoparse(updated_end).year > 1970:
```

Can we add a validation statement to make sure `update_start` is not before `update_end`?

If it is, maybe return an error message?

---

We already have that validation; it happens inside `_build_modified_on_filter` here: https://github.com/RedHatInsights/insights-host-inventory/blob/sp-update/api/filtering/filtering.py#L314